docs/videos/healthcare-and-life-sciences/README.md

```diff
@@ -2,7 +2,7 @@
 
 The Healthcare and Life Science (HCLS) [blueprint](./hcls-blueprint.yaml) in
 this folder captures an advanced architecture that can be used to run GROMACS
-with GPUs on Google Cloud.
+with GPUs or CPUs on Google Cloud.
 
 ## Getting Started
```

Comment:
The water benchmark example link is broken
[Water Benchmark Example](#water-benchmark-example-instructions) -> [Water Benchmark Example](#water-benchmark-example)